starter-code/scripts/article.js

```diff
@@ -50,13 +50,30 @@ Article.fetchAll = function() {
     // When rawData is already in localStorage,
     // we can load it with the .loadAll function above,
     // and then render the index page (using the proper method on the articleView object).
-    Article.loadAll(); //TODO: What do we pass in to loadAll()?
+    Article.loadAll(JSON.parse(localStorage.getItem('data'))); //TODO: What do we pass in to loadAll()?
```

Comment:
I figured out why your code does not work on the initial load. Look at this line. You are getting from localStorage something called data. But if you look above at the if statement, you are looking for something in local storage called rawData. So if you are pushing and pulling data from local storage but your initial check is for rawData, these will never line up. We don't want to modify the existing code(code we gave you), so how can you change your code to match our code? Let me know if you need help with this!